pysatNASA/instruments/methods/cdaweb.py

```diff
@@ -1,7 +1,9 @@
 # -*- coding: utf-8 -*-
-"""Provides default routines for integrating NASA CDAWeb instruments into
-pysat. Adding new CDAWeb datasets should only require mininal user
-intervention.
+"""Provides default routines for NASA CDAWeb instruments into pysat.
```

Comment:
Few residual `string` in docstring. Also some residual `array_like` instead of `array-like`.